virttest/env_process.py

```diff
@@ -1240,6 +1240,25 @@ def preprocess(test, params, env):
             test.cancel("Got host qemu version:%s, which is not in %s" %
                         (host_qemu, required_qemu))
 
+    # Checking required virtio-win version, if not satisfied, cancel test
+    if params.get("required_virtio_win"):
+        if params.get("cdrom_virtio"):
+            required_virtio_win = params.get("required_virtio_win")
+            LOG.info("Test requires virtio-win version: %s" % required_virtio_win)
+            cdrom_virtio = params["cdrom_virtio"]
+            if re.search('prewhql', cdrom_virtio):
+                match = re.search("\d+.\d+-\d+", cdrom_virtio)
+            else:
+                match = re.search("\d+.\d+\d+-\d+", cdrom_virtio)
+            if match is None:
+                test.cancel("Can not get virtio-win version")
```

Comment:
1. I would suggest here use `error` instead of `cancel` as here must be something wrong, not only a simple precondition mismatch (where cancel suite for).
2. Print the cdrom_virtio detail here would be more friendly for debugging.
```suggestion
                test.error("Can not get virtio-win version from \"cdrom_virtio\": %s" % cdrom_virtio)
```